test/e2e/tests/request-queuing/multi-dapp-sendTx-revokePermission.spec.js

```diff
@@ -48,14 +45,11 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks revok
         await openDapp(driver, undefined, DAPP_URL);
 
         // Connect to dapp 1
-        await driver.findClickableElement({ text: 'Connect', tag: 'button' });
-        await driver.clickElement('#connectButton');
+        await driver.clickElement({ text: 'Connect', tag: 'button' });
 
-        await driver.delay(regularDelayMs);
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
-        await switchToNotificationWindow(driver);
```

Comment:
instead of this, we can use the new method for switching using window title.
then we can click element and wait until the popup is closed before proceeding